### PR TITLE
Fixed docker issues

### PR DESCRIPTION
**Pull request by vol4onok**

### CODE REVIEW

**Review by Kyrylo-Meretskyi — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 The Spryker SDK aims to provide a single entry point to accelerate your productivity working with Spryker.
 No matter if you want to validate existing code, implement new features with Spryker or go live with your project,
 the Spryker SDK provides you the tools to fulfill your mission faster and cut out the boring parts of the development,
-so you can focus developing exciting features for your business case.
+so you can focus developinPHP_INI_SCAN_DIRg exciting features for your business case.
```
Comment (miroshnikov-spryker) [inline README.md:10]: Accidental ctrl+v

File: bin/spryker-sdk.sh [hunk 2]
```diff
@@ -24,23 +24,23 @@ myIp='host.docker.internal'
...
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
+export EXECUTABLE_FILE_PATH=$(realpath "$0")
 
 case $MODE in
 "debug")
   echo "Ensure mutagen is running by executing: mutagen compose -f docker-compose.yml -f docker-compose.dev.yml up -d"
   export SPRYKER_XDEBUG_HOST_IP=${myIp}
   export PHP_IDE_CONFIG=serverName=spryker-sdk
-  docker-compose -f "${SDK_DIR}/docker-compose.yml" -f "${SDK_DIR}/../docker-compose.dev.yml" run --rm -e XDEBUG_SESSION=1 -e APP_ENV=dev spryker-sdk "$ARGUMENTS"
```
Comment (miroshnikov-spryker) [inline bin/spryker-sdk.sh:38]: I think the command should use docker-compose._debug_.yml same as build/bin/spryker-sdk.sh

File: src/Presentation/Console/Commands/UpdateCommand.php [hunk 3]
```diff
@@ -125,7 +125,7 @@ protected function checkForUpdate(OutputInterface $output)
 
         $latestVersion = $this->getLatestVersion($output);
 
-        if (version_compare($currentVersion, $latestVersion) === -1) {
+        if (version_compare($currentVersion, $latestVersion, '>')) {
```
Comment (miroshnikov-spryker) [inline src/Presentation/Console/Commands/UpdateCommand.php:128]: Shouldn't there be `<` sign so the condition would be `if ($currentVersion < $latestVersion)` ?

Comment (vol4onok) [reply src/Presentation/Console/Commands/UpdateCommand.php:128]: the latest version is what we had, the current version is what we have now.

**Pull request merged**